waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -96,3 +98,18 @@ generatePrismaClient genProjectRootDirAbs = do
       writeDbSchemaChecksumToFile genProjectRootDirAbs (SP.castFile dbSchemaChecksumOnLastGenerateFileProjectRootDir)
       return $ Right ()
     ExitFailure code -> return $ Left $ "Prisma client generation failed with exit code: " ++ show code
+
+-- | Checks `prisma migrate diff` exit code to determine if schema.prisma is
+-- different than the DB. Returns Nothing on error as we do not know the current state.
```

Comment:
Aha hmmmm, so Just True means schema is in sync with db, but what is the difference between Just False and Nothing? So for Just False we know it is not in sync, but for Nothing we actually haven't learned anything, is that so? So Nothing means we don't know if it is in sync or not? Why would that even happen?

Reply:
Just False means we know they are not in sync. Nothing means some error running the command, so we don't know what is what.

Reply:
Ok! I think y ou actually mentioned in another place the cause of this could be DB that is offline, wrong DB connection URL, and similar.
Could we comment on this a bit more, here in the header comment of this function?
Smth like:
> Returns Just True if schema.prisma is same as DB, Just False if it is different, and Nothing if the check itself failed, which could e.g. happen if connection to the DB could not be established.

Reply:
Yep, sounds good! 👍🏻